File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/non-plagiarized/11/T04.kt
package IR

/**
 *
 * @author B00A50C448238A71ED479F81FA4D9066
 */

import java.text.DecimalFormat

fun main() {
    val d = DecimalFormat("0.000")
    
    println("Miles\tKilometers")
    for (i in 1..10) {
        println("$i\t${d.format(i * 1.609)}")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/non-plagiarized/11/T04.ts
import { DecimalFormat } from 'java.text';

function main(): void {
    const d: DecimalFormat = new DecimalFormat('0.000');

    console.log('Miles\tKilometers');
    for (let i = 1; i <= 10; i++) {
        console.log(`${i}\t${d.format(i * 1.609)}`);
    }
}

main();

